src/lib/pages/backoffice/DocumentRequest/DocumentRequestDetails/DocumentRequestSteps/ProviderStep/ProviderStep.js

```diff
@@ -82,14 +82,7 @@ class AcqProvider extends Component {
             <ESSelector
               icon={<AcquisitionOrderIcon />}
               onSelectResult={this.onSelectResult}
-              query={() =>
-                orderApi.list(
-                  orderApi
-                    .query()
-                    .withState('PENDING')
-                    .qs()
-                )
-              }
+              query={orderApi.listWithPendingStatus}
```

Comment:
I'm not sure if supplying only the `PENDING` ones will be OK in this case. @topless @ntarocco is there any use case we would like to connect orders/ILLs in the states different than `PENDING`?

Reply:
Maybe it will be safer to get the orders that do not have yet a `document_request_pid`, independently of the state.
Imagine the scenario where the librarian will not follow the workflow and:
1. see the doc request
2. go to the order page and create a new order
3. come back to the doc request and want to attach it.

WDYT?

Reply:
There is no `document_request_pid` in the order and there is no connection between the order and the document request from the order's side.